pallets/migrations/src/tests.rs

```diff
@@ -226,14 +221,6 @@ fn migration_should_only_be_invoked_once() {
 	);
 }
 
-#[test]
```

Comment:
Why did this test dissapear?

Reply:
Because this test is not relevant anymore, I should replace it

Reply:
I replaced this test by a new one named `on_runtime_upgrade_charges_max_block_weights`